wire-ios/Wire-iOS/Sources/AppDelegate.swift

```diff
@@ -268,11 +268,11 @@ private extension AppDelegate {
             fatalError("sessionManager is not created")
         }
 
-        let navigator = Navigator(NoBackTitleNavigationController())
-        appRootRouter = AppRootRouter(viewController: viewController,
-                                      navigator: navigator,
-                                      sessionManager: sessionManager,
-                                      appStateCalculator: appStateCalculator)
+        appRootRouter = AppRootRouter(
+            viewController: viewController,
```

Comment:
not sure we keep the functonality to have no back button title here

Reply:
I'm not sure what you mean. If I'm not mistaken, I just removed dead code.

Reply:
I think you're removing the `NoBackTitleNavigationController` which is a subclass of UINavigationController  that would hide the title in back button.

Reply:
ok bye bye deadcode